fields/types/list/ListField.js

```diff
@@ -0,0 +1,141 @@
+/* eslint-disable react/jsx-no-bind */
+
+import assign from 'object-assign';
+import { css, StyleSheet } from 'aphrodite/no-important';
+import React from 'react';
+import Field from '../Field';
+import Domify from 'react-domify';
+
+import { Fields } from 'FieldTypes';
+import { Button, GlyphButton } from '../../../admin/client/App/elemental';
+import InvalidFieldType from '../../../admin/client/App/shared/InvalidFieldType';
+
+let i = 0;
+function generateId () {
+	return i++;
+};
+
+const ItemDom = ({ name, id, onRemove, children }) => (
+	<div style={{
+		borderTop: '2px solid #eee',
+		paddingTop: 15,
+	}}>
+		{name && <input type="hidden" name={name} value={id}/>}
+		{children}
+		<div style={{ textAlign: 'right', paddingBottom: 10 }}>
+			<Button size="xsmall" color="danger" onClick={onRemove}>
+				Remove
+			</Button>
+		</div>
+	</div>
+);
+
+module.exports = Field.create({
+	displayName: 'ListField',
+	statics: {
+		type: 'List',
+	},
+	propTypes: {
+		fields: React.PropTypes.object.isRequired,
+		label: React.PropTypes.string,
+		onChange: React.PropTypes.func.isRequired,
+		path: React.PropTypes.string.isRequired,
+		value: React.PropTypes.array,
```

Comment:
perhaps this comment is the wrong way to request this, but I wanted to have visibility on this:
it would be very nice to have a customizable limit on the amount of items in a list, say "max: 10" i.e. by default. Thoughts?